templates/drupal/0/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  web:
+    image: drupal:${drupal_tag}
```

Comment:
@slimatic Thanks for your contribution to the community catalog, the template reviewed and tested, the only note is that you can use the [versioning system](http://docs.rancher.com/rancher/v1.2/en/catalog/private-catalog/#directory-structure) for creating custom catalogs instead of using a variable to specify the version for drupal and mysql,